draft/2023-08-23-this-week-in-rust.md

```diff
@@ -43,6 +43,12 @@ and just ask the editors to select the category.
 
 ### Miscellaneous
 
+- [video] [I failed to build multiplayer pong in Rust](https://www.youtube.com/watch?v=wpx9qhKEuP8)
+- [video] [A useful tip for every (?) Rust code base](https://www.youtube.com/watch?v=FJsBGiRTM9A)
+- [video] [What Rust Discovered](https://www.youtube.com/watch?v=v6RxJsk8otY)
+- [video] [Delightful command-line utilities with Rust](https://www.youtube.com/watch?v=Y-LTWNciEks)
+- [video] [Rust 1.71.0](https://www.youtube.com/watch?v=H8aHX-JkiZs)
```

Comment:
```suggestion
- [video] [Delightful command-line utilities with Rust](https://www.youtube.com/watch?v=Y-LTWNciEks)
```

Reply:
hi @c-git! We've discussed internally and cut down this submission by a little bit to ensure that the amount of content that we provide to readers isn't too high, and is extremely useful. We tend to avoid things that are dated or in a listicle format especially. Thank you so much for your submission and for continuing to bring new content to the newsletter!

Reply:
No problem, thank you for the feedback. I'll try to bear that in mind.